pom.xml

```diff
@@ -106,7 +106,7 @@
               <overwrite>true</overwrite>
               <resources>
                 <resource>
-                  <directory>${project.basedir}/configs/odoo_config</directory>
+                  <directory>${project.basedir}/configs/odoo</directory>
```

Comment:
There can be additional types of configurations for Odoo, so it would be reasonable to allow for this, such as:
`directory>${project.basedir}/configs/odoo/initializer</directory>`

Reply:
Done for odoo and openmrs configs.